opennav_coverage/CMakeLists.txt

```diff
@@ -19,7 +20,7 @@ find_package(Fields2Cover REQUIRED)
 
 # potentially replace with nav2_common, nav2_package()
 set(CMAKE_CXX_STANDARD 17)
-add_compile_options(-Wall -Wextra -Wpedantic -Werror -Wdeprecated -fPIC -Wshadow -Wnull-dereference)
+# add_compile_options(-Wall -Wextra -Wpedantic -Werror -Wdeprecated -fPIC -Wshadow -Wnull-dereference)
```

Comment:
Add comment: "These are commented out due to quality issues compiling F2C"

Reply:
oh, I can revert that since switching to v1.2.1 didn't solve the issue anyway.